Cargo.toml

```diff
@@ -1,28 +1,29 @@
 [workspace]
 resolver = "2"
 members = [
-  "bonsai/sdk",
-  "risc0/binfmt",
-  "risc0/build",
-  "risc0/build_kernel",
-  "risc0/cargo-risczero",
-  "risc0/circuit/recursion",
-  "risc0/circuit/recursion-sys",
-  "risc0/circuit/rv32im",
-  "risc0/circuit/rv32im-sys",
-  "risc0/core",
-  "risc0/groth16",
-  "risc0/r0vm",
-  "risc0/sys",
-  "risc0/tools",
-  "risc0/zkp",
-  "risc0/zkvm",
-  "risc0/zkvm/methods",
-  "risc0/zkvm/platform",
-  "risc0/zkvm/receipts",
-  "tools/hotbench",
-  "website/doc-test/main",
-  "xtask",
+    "bonsai/sdk",
+    "risc0/binfmt",
+    "risc0/build",
+    "risc0/build_kernel",
+    "risc0/cargo-risczero",
+    "risc0/circuit/recursion",
+    "risc0/circuit/recursion-sys",
+    "risc0/circuit/rv32im",
+    "risc0/circuit/rv32im-sys",
+    "risc0/core",
+    "risc0/groth16",
+    "risc0/r0vm",
+    "risc0/sys",
+    "risc0/tools",
+    "risc0/zkp",
+    "risc0/zkvm",
+    "risc0/zkvm/methods",
+    "risc0/zkvm/platform",
+    "risc0/zkvm/receipts",
+    "rzup",
+    "tools/hotbench",
+    "website/doc-test/main",
+    "xtask",
```

Comment:
Not sure where these whitespace changes are coming from. I think my editor (vscode) follows the `.editorconfig` which defaults to `indent_size = 2`

Reply:
I noticed that my editor was not following that - I'll change it back